kornia/sensor/camera/camera.py

```diff
@@ -0,0 +1,110 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import Optional, Tuple
+
+from kornia.core import Module, Tensor, stack, zeros_like
+from kornia.core.check import KORNIA_CHECK_TYPE
+from kornia.geometry.vector import Vector2, Vector3
+from kornia.sensor.camera.distortion import AffineTransform, CameraDistortionType
+from kornia.sensor.camera.projection import CameraProjectionType, Z1Projection
+
+
+@dataclass
+class ImageSize:
+    height: int
+    width: int
+
+
+class CameraModel(Module):
```

Comment:
if it's a Module should we register the params as a nn.Parameter ?

Reply:
make sense to add the Pose params as nn.Parameter not sure if we want to add the other params.